### DISCUSSION TITLE
[Finnix 125] Default login and password

### DISCUSSION THREAD

**Discussion by ilyaprokopenko** [finnix/finnix#45] [Q&A]
Hello everyone! When I tried  to boot from LiveCD and select first menu point  I see request for login and  password afrer system start. I nothing found in documentation and  by Google about login/password for Finnix.  Could  you help me with information about default root login and password?
Thank you for help.
![image](https://github.com/finnix/finnix/assets/133878112/f93cdfc3-a169-4229-9350-a32ce60fd623)

**Comment by alzwded** [comment]
(Just a regular user here) I grabbed finnix 125 from finnix.org and started a VM and it booted to a root prompt normally. By default, there is only root and it does not have a password (unless you added kernel params). Switching VTYs also gets you root logged in and at a prompt. I'm not sure what you did to set up your "LiveCD" there. Or maybe the uploaded iso was borked when you downloaded it.
